### ISSUE TITLE
Test OCAP2 as replacement for CNTR

### ISSUE THREAD

**Issue by enrico-ghidoni** [CntoDev/central-plaza#19]
Opening a discussion to set up an [OCAP2](https://github.com/OCAP2/OCAP) istance on the community infrastructure to test it out and evaluate whether it's a good replacement for the internally developed (and unmaintained) [CNTR](https://github.com/CntoDev/CNTR) that is currently in use. @CntoDev/rnd and @CntoDev/ssgt please share your two cents on the topic.

Concerns to address include:
 - are all the maps currently supported by CNTR also available for OCAP2? (Maps list available [here](https://drive.google.com/drive/folders/105Rm3clVNVQsJzd_74Ub_NwND4ObvHAW)
 - are CNTR recordings compatible with OCAP? If not, is there a way to make them compatible? If not, what do we do?
 - where do we deploy the frontend part? Currently CNTR's frontend is deployed on `Main Server` but since OCAP2 provides a Docker image for that it could be possible to deploy its frontend on `Tools Server` to further consolidate it as secondary services hosting solution and to remove some (light) strain on `Main Server`

**Comment by enrico-ghidoni** [comment]
Just to clarify, the idea is to deploy OCAP2 **alongside with** CNTR and reach out for community feedback on which tool is better to keep.

**Comment by Shiny-CNTO** [comment]
Looks very easy to set up. The only thing is if the files are compatible.
We can choose what server to run on. If we can make some sort of automation that the files needed for an OCAP replay to work can be auto send to linux we can even run it on there using the docker image. If we can't setup that automation then Windows might be easier.
Otherwise no issues from the infra side.

**Comment by JamesTheClarke** [comment]
> Just to clarify, the idea is to deploy OCAP2 **alongside with** CNTR and reach out for community feedback on which tool is better to keep.

Correct, we will deploy OCAP2 asap for testing purposes, however a complete rollout or potential switch from CNTR to OCAP2 will require a Staff Forum / Monthly Meeting discussion (I will set up a Staff Forum thread in advance to keep the community informed).

### Questions to be answered during testing:

- [ ] Is there a noticeable performance impact when recording sessisions comparing **OCAP2** with **CNTR**?
- [ ] Is there a noticeable performance impact when recording sessisions comparing **OCAP2** on it's own with **OCAP2 plus [OCAP2Arma mod**](https://steamcommunity.com/sharedfiles/filedetails/?id=2713040198) (which enables 3d playback of sessions similar to how ShackTac records their sessions)?
- [ ] Are existing CNTR files able to be made compatible with OCAP2?

Personally, I see the last question as less of an issue because we could always run CNTR as a legacy application (not the session recording, just the playback functionality) on our website.
